apps/web/src/app/[locale]/challenge/[slug]/solutions/_components/index.tsx

```diff
@@ -144,10 +146,12 @@ function SolutionRow({
   solution: NonNullable<PaginatedSolution['sharedSolution']>[number];
 }) {
   const { slug } = useParams();
+  const queryString = useGetQueryString();
   return (
     <Link
+      href={`/challenge/${slug}/solutions/${solution.id}?${queryString}`}
       className="flex cursor-pointer flex-col gap-2 p-4 duration-300 hover:bg-neutral-100 dark:hover:bg-zinc-700/50"
-      href={`/challenge/${slug}/solutions/${solution.id}`}
+      // href={}
```

Comment:
remove

Reply:
Fixed
Sorry I didn't check this beforehand
Thanks for review